about.html

```diff
@@ -13,61 +13,68 @@
-
-
-    <img src="img/about-plan.png" alt="strategy">
-    
-    Strategy
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
-
-
-    <img src="img/about-working.png" alt="strategy">
-    
-    How We Work
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
-
-
-    <img src="img/about-office.png" alt="strategy">
-    
-    Places We Work
-
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
-    <img src="img/about-meeting.png" alt="strategy">
-    
-    Collaboration
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
-    Let's Work Together
-
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
-        
-      <footer>
-        <nav>
-          <a href="index.html">Home</a>
-          <a href="#">About</a>
-          <a href="#">Products</a>
-          <a href="#">Blog</a>
-          <a href="#">Contact</a>
-        </nav>
-      </footer>
-    </div><!-- container -->        
+        <header>
+            <img src="img/lambda-black.png" alt="lambda logo">
+            <nav>
+                <a href="index.html">Home</a>
+                <a href="about.html">About</a>
+                <a href="#">Products</a>
+                <a href="#">Blog</a>
+                <a href="#">Contact</a>
+            </nav>
+        </header>
+
+        <img class="under-nav" src="img/jumbo-about.png" alt="daytime city skyline">
+
+        <section class= "text-container">
+          <h2>About</h2>
+          <p>Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.</p>
+        </section>
+   
+        <section class="middle-content">
+          <section class="boxes">
+              <section class="about-box">
+                  <img src="img/about-plan.png" alt="strategy">
+                  <h2>Strategy</h2>
+                  <p>Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.</p>
+                  <button>Learn More</button>
+              </section>
+              <section class="about-box">
+                  <img src="img/about-working.png" alt="strategy">
```

Comment:
I know the code given already had the `alt` attributes written but a good habit would be to write something more descriptive than `strategy` as this attribute can be used by users that can not necessarily see the screen.